apps/web/components/booking/BookingListItem.tsx

```diff
@@ -280,7 +280,7 @@ function BookingListItem(booking: BookingItemProps) {
         isOpenDialog={isOpenSetLocationDialog}
         setShowLocationModal={setIsOpenLocationDialog}
       />
-      {booking.paid && (
+      {booking.paid && booking?.payment[0] && (
```

Comment:
```suggestion
      {booking.paid && booking.payment[0] && (
```

Reply:
We can make the similar change at other places.

Reply:
Shouldn't it be added with this condition?
`booking.paid && booking?.payment[0]?.paymentOption === "HOLD"
as per `https://github.com/calcom/cal.com/blob/d88a2bba0df1f08f6e2fa8c01a3b22a8cd24fd4d/apps/web/components/booking/BookingListItem.tsx#L465

Though I feel that we can simply remove this condition(which makes things complex), as Dialog won't be shown anyway unless the CTA is there to be clicked.